interactions/models/discord/channel.py

```diff
@@ -2731,4 +2737,5 @@ def process_permission_overwrites(
     ChannelType.DM: DM,
     ChannelType.GROUP_DM: DMGroup,
     ChannelType.GUILD_FORUM: GuildForum,
+    ChannelType.MEDIA_CHANNEL: GuildMediaChannel,
```

Comment:
This doesn't seem to follow our current naming conventions, would something like this be better fitting?

```suggestion
    ChannelType.GUILD_MEDIA: GuildMedia,
```

Reply:
Sure - cant merge this suggestion as this is a widespread change